develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+# Selecting Entities Using the Item Selector API [](id=selecting-entities-using-the-item-selector-api)
```

Comment:
@mwilliams2014 @sergiogonzalez @robertoDiaz - 
This tutorial is really about _providing_ selectable entities. I think this title would be more appropriate: _Providing Selectable Entities_

Thoughts?